packages/cli/src/cmds/agentInstaller/agentProcedure.ts

```diff
@@ -251,7 +252,14 @@ export default abstract class AgentProcedure {
         UI.error(addGitReturn.errorMessage);
       }
 
-      const commitGitReturn = await this.gitCommit(filesDiff, 'feat: Install AppMap.');
+      const commitGitReturn = await this.gitCommit(filesDiff, `feat: Configure AppMap for this project
```

Comment:
I think we should drop the "feat: " part. It would be confusing to any team that doesn't regularly use it.

Reply:
Dropped.